source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
-This guide shows you how to create an application that uses the Node.js
-driver to connect to a MongoDB Atlas cluster.
+*Estimated time to complete this guide: 35 minutes.*
 
-Check for Node and NPM
-----------------------
+This guide shows you how to create an application that uses the
```

Comment:
Do we need a title here? "Overview" etc.? Seems a bit strange to have this just floating between the "On this page" box and the "Set up Your Project" section.

Reply:
Was there feedback about that? We did not have a title for the lead-in sentence before (see the original ll. 13-14)

Reply:
Not sure if there was previous feedback by others on this. Perhaps it's because the "On this page" box wasn't here in Nathan's version? I'm flexible on this given that we'll be removing the estimated completion time sentence. It seemed like Michael Waltzer had feedback on the inclusion of the "On this page" box, but I can't add him to GitHub apparently.